controller/src/util/node_watcher.rs

```diff
@@ -408,12 +410,16 @@ mod tests {
         let _ = env_logger::builder().is_test(true).try_init();
 
         let node_json = file::read_file_to_string("../test/json/node-b-not-ready.json");
-        let node: NodeObject = serde_json::from_str(&node_json).unwrap();
+        let node: Node = serde_json::from_str(&node_json).unwrap();
         let mut node_watcher = NodeWatcher::new();
+        // Add Node to known_nodes with original Running state
+        node_watcher
```

Comment:
why do we need to add a known_node now but didn't before?

Reply:
Good point. this was left over from when i was trying to use the `watcher` instead of `Api::watch`

Reply:
removed in 67595e71132f